pkgs/development/libraries/kf5gpgmepp/cmake-minimum-required.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0a86f83..2b91020 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,6 +1,6 @@
+-cmake_minimum_required(VERSION 2.8.12)
++cmake_minimum_required(VERSION 3.10)
+ 
+-project(Gpgmepp)
++project(Gpgmepp VERSION 16.08.3)
```

Comment:
If the package is updated in the future without a cmake fix, will this cause an incorrect version to appear anywhere?